snapcraft/cli/help.py

```diff
@@ -56,7 +57,16 @@ def help_command(topic, devel):
         snapcraft help sources
         snapcraft help go
     """
-    if topic == 'topics':
+    if not topic:
+        click.echo(ctx.parent.get_help())
+        click.echo(dedent("""\
+
+            You can obtain more help by running
```

Comment:
On my pr about commands on the error message, I tried to make them consistent using a wording like this:
`To obtain more help, run`

Reply:
sure thing

Reply:
oh wait, this is not an error message

Reply:
I know this one is not an error message, but I would like us to use consistent wording when asking or suggesting the user to run a command.